src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -45,12 +45,6 @@ public TreeSubSet(SortedSet<T> Underlying, T Min, T Max, bool lowerBoundActive,
                 VersionCheckImpl();
             }
 
-            private TreeSubSet(SerializationInfo info, StreamingContext context)
```

Comment:
I agree we don't have that data. As a proxy, how's this? It looks like at least on NetFX, the only way to get a TreeSubSet is through GetViewBetween. According to apisof.net, GetViewBetween has very little usage. Additionally, serializing SortedSets is pretty rare compared to other concrete collections. The intersection of those two very small percentages is hopefully ~zero.